index.html

```diff
@@ -196,5 +195,47 @@ <h2>Skills</h2>
             </div>
         </div>
     </section>
+    <section class="contact">
+            <h2 class="contact-header">I'm always<br>interested in <br>
+                hearing about <br>new projects, so <br>
+                if you'd like to<br> chat please get<br>in touch.
+            </h2>
+        <form action="https://formspree.io/f/mlezlkwz" method="post">
+            <ul>
+            <li><input type="text" name="fullname" placeholder="Full name"
+                maxlength="30" required></li>
+            <li><input type="email" name="email" placeholder="Email address" required></li>
+            <li><textarea name="message" id="form-msg" cols="30" rows="10"
+                placeholder="Enter text here" maxlength="300" required></textarea></li>
```

Comment:
- [ ] The `maxlength` attribute for your textarea should be **500**, not 300. Kindly fix this.